bundletool/bundletool.go

```diff
@@ -0,0 +1,115 @@
+package bundletool
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
+)
+
+// KeystoreConfig represents the parameters required to sign an APK.
+type KeystoreConfig struct {
+	// Specifies the path to the deployment keystore used to sign the APKs.
+	Path string
+	// If you’re specifying a password in plain text, qualify it with pass:.
+	// If you’re passing the path to a file that contains the password, qualify it with file:.
+	KeystorePassword string
+	// Specifies the alias of the signing key you want to use.
+	SigningKeyAlias string
+	// If you’re specifying a password in plain text, qualify it with pass:.
+	// If you’re passing the path to a file that contains the password, qualify it with file:.
+	SigningKeyPassword string
+}
+
+const (
+	githubReleaseBaseURL = "https://github.com/google/bundletool/releases/download"
+	bundletoolAllJarName = "bundletool-all.jar"
+)
+
+// Tool represent a wrapper around the bundletool.
+type Tool struct {
+	path string
+}
+
+// New downloads the bundletool executable from Github and places it to a temporary path.
+func New(version string) (*Tool, error) {
+	tmpPth, err := pathutil.NormalizedOSTempDirPath("tool")
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := getFromMultipleSources([]string{
+		githubReleaseBaseURL + "/" + version + "/" + "bundletool-all-" + version + ".jar",
```

Comment:
i suggest using this package: https://github.com/bitrise-io/go-utils/blob/master/urlutil/urlutil.go#L10